### ISSUE TITLE
Installed exception on webstorm load V1.80

### ISSUE THREAD

**Issue by jjarrell50** [vsch/idea-multimarkdown#268]
1.7 Null exception on startup.
```
null
java.lang.NullPointerException
	at com.vladsch.idea.multimarkdown.editor.MultiMarkdownToHtmlSerializer.<init>(MultiMarkdownToHtmlSerializer.java:81)
	at com.vladsch.idea.multimarkdown.editor.resources.JavaFxHtmlGenerator.toHtml(JavaFxHtmlGenerator.kt:60)
	at com.vladsch.idea.multimarkdown.editor.MarkdownPreviewFileEditor.makeHtmlPage(MarkdownPreviewFileEditor.kt:44)
	at com.vladsch.idea.multimarkdown.editor.PreviewFileEditorBase.updateHtmlRunner(PreviewFileEditorBase.kt:448)
	at com.vladsch.idea.multimarkdown.editor.PreviewFileEditorBase.access$updateHtmlRunner(PreviewFileEditorBase.kt:57)
	at com.vladsch.idea.multimarkdown.editor.PreviewFileEditorBase$updateHtml$$inlined$synchronized$lambda$1.run(PreviewFileEditorBase.kt:404)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:238)
	at com.intellij.util.Alarm$Request$1.run(Alarm.java:378)
	at com.intellij.util.Alarm$Request.run(Alarm.java:398)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at com.intellij.util.concurrency.SchedulingWrapper$MyScheduledFutureTask.run(SchedulingWrapper.java:227)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at com.intellij.ide.IdeEventQueue.b(IdeEventQueue.java:857)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:658)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:386)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

1.8
`update failed for AnAction with ID=ProjectViewPopupMenuRefactoringGroup: com/intellij/psi/PsiJavaFile`

```
update failed for AnAction with ID=ProjectViewPopupMenuRefactoringGroup: com/intellij/psi/PsiJavaFile
java.lang.NoClassDefFoundError: com/intellij/psi/PsiJavaFile
	at com.vladsch.idea.multimarkdown.psi.flexmark.FlexmarkExampleOptionLiteralRenameHandler.isAvailableOnDataContext(FlexmarkExampleOptionLiteralRenameHandler.java:44)
	at com.intellij.refactoring.rename.RenameHandlerRegistry.hasAvailableHandler(RenameHandlerRegistry.java:64)
	at com.intellij.refactoring.actions.RenameElementAction.isEnabledOnDataContext(RenameElementAction.java:64)
	at com.intellij.refactoring.actions.BaseRefactoringAction.update(BaseRefactoringAction.java:172)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:120)
	at com.intellij.openapi.actionSystem.ActionGroupUtil.isActionEnabledAndVisible(ActionGroupUtil.java:103)
	at com.intellij.openapi.actionSystem.ActionGroupUtil.isGroupEmpty(ActionGroupUtil.java:48)
	at com.intellij.openapi.actionSystem.ActionGroupUtil.isGroupEmpty(ActionGroupUtil.java:50)
	at com.intellij.openapi.actionSystem.ActionGroupUtil.isGroupEmpty(ActionGroupUtil.java:39)
	at com.intellij.ide.actions.NonTrivialActionGroup.update(NonTrivialActionGroup.java:37)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:120)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:188)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:139)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:86)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenu(Utils.java:280)
	at com.intellij.openapi.actionSystem.impl.ActionPopupMenuImpl$MyMenu.show(ActionPopupMenuImpl.java:95)
	at com.intellij.ide.ui.customization.CustomizationUtil$3.invokePopup(CustomizationUtil.java:291)
	at com.intellij.ui.PopupHandler.mouseReleased(PopupHandler.java:55)
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:290)
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:289)
	at java.awt.Component.processMouseEvent(Component.java:6535)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at com.intellij.ui.treeStructure.Tree.processMouseEvent(Tree.java:410)
	at com.intellij.ide.dnd.aware.DnDAwareTree.processMouseEvent(DnDAwareTree.java:58)
	at java.awt.Component.processEvent(Component.java:6300)
	at java.awt.Container.processEvent(Container.java:2236)
	at java.awt.Component.dispatchEventImpl(Component.java:4891)
	at java.awt.Container.dispatchEventImpl(Container.java:2294)
	at java.awt.Component.dispatchEvent(Component.java:4713)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4888)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4525)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4466)
	at java.awt.Container.dispatchEventImpl(Container.java:2280)
	at java.awt.Window.dispatchEventImpl(Window.java:2750)
	at java.awt.Component.dispatchEvent(Component.java:4713)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at com.intellij.ide.IdeEventQueue.b(IdeEventQueue.java:857)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:654)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:386)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.ClassNotFoundException: com.intellij.psi.PsiJavaFile PluginClassLoader[com.vladsch.idea.multimarkdown, 1.8.0]
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:64)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 56 more
```

**Comment by vsch** [comment]
@jjarrell50, oops. It is caused by some code expecting Java support. I will fix it ASAP and post an updated release here.

**Comment by vsch** [comment]
@jjarrell50, thank you for bringing this to my attention. I tested the plugin on PhpStorm which also has no Java support and thought it covered but missed the refactoring part. This exception would be caused on any refactoring action in an IDE without Java. 😨 

It  is bad enough that I had to make a new release with a fix, 1.8.1. You can download it from the official site or here: https://github.com/vsch/idea-multimarkdown/raw/master/dist/idea-multimarkdown.1.8.1.zip
